plugins/lookup/aws_secret.py

```diff
@@ -35,6 +39,7 @@
     description:
         - Join two or more entries to form an extended secret.
         - This is useful for overcoming the 4096 character limit imposed by AWS.
+        - No effect when used with `bypath`
```

Comment:
```suggestion
        - No effect when used with I(bypath).
```

Reply:
Similarly it would be good if you could update this on line 23: `I(version_id) and I(version_stage)`